ErpNet.FP.Server/wwwroot/app.js

```diff
@@ -72,6 +77,38 @@ function showAvailablePrinters() {
                 }
                 paymentMappingsContent += '</div>';
 
+                var printerOpionsContent = '';
+                if (printerProps) {
+                    for (var i = 0; i < availablePrinterOptions.length; i++) {
+                        var po = availablePrinterOptions[i];
+                        var optionAvailable = printerProps.printerOptions[po];
+                        if (optionAvailable) {
+                            var printerOptionName = useOfPrinterOptions[i];
+                            var printerOptionHint = hintsForPrinterOptions[i];
+                            var printerOptionDisplayText = displayTextsForPrinterOptions[i];
+                            var inputId = printer.serialNumber + '_po_' + printerOptionName;
+                            var printerOptionValue = '';
+
+                            var pv = printerProps.printerOptions[printerOptionName]
+                            //if (pv) {
+                                if (pv == 'True')
+                                    printerOptionValue = 'checked';
+                            //}
+
+                            printerOpionsContent +=
+                                '<label title ="' + printerOptionHint + '" for="' + inputId + '" style="margin-right: 0;">' +
```

Comment:
I guess the index.css is some version of bootstrap; I guess bootstrap v4.

You could use spacing [utilities instead](https://getbootstrap.com/docs/4.6/utilities/spacing/)

e.g. class="ml-0"